app/services/google_integration/classroom/parsers/students.rb

```diff
@@ -34,6 +34,7 @@ def self.parse_hash(hash)
     profile_data = hash['profile']
     {
       name: profile_data['name']['fullName'],
+      first_name: profile_data['name']['givenName'],
```

Comment:
I don't know why this was added here -- ostensibly Marcello was just doing a Google integration bugfix.